src/Features/Core/Portable/QualifyMemberAccess/AbstractQualifyMemberAccessDiagnosticAnalyzer.cs

```diff
@@ -152,7 +152,8 @@ private bool IsStaticMemberOrTargetMethod(IOperation operation)
             switch (operation)
             {
```

Comment:
You may also want rename the method to indicate it returns true for all local functions, i.e. say `IsMemberOrTargetMethodStaticOrLocalFunction`

Reply:
Got it, just renamed.